### PR TITLE
Update Presentation API samples

### PR DESCRIPTION
**Pull request by beaufortfrancois**
I've spent some time updating our Presentation API samples to make them easier to read.
WDYT @mfoltzgoogle?

Live previews:
https://beaufortfrancois.github.io/samples/presentation-api/
https://beaufortfrancois.github.io/samples/presentation-api/cast.html

### CODE REVIEW

File: presentation-api/receiver/receiver.js [hunk 1]
```diff
@@ -1,65 +1,62 @@
...
-      maybeSetFruit(messageObj.message.toLowerCase());
-      connection.send('Received message ' + messageIdx);
-    };
+  connection.addEventListener('message', function(event) {
+    messageIdx++;
+    const data = JSON.parse(event.data);
+    const logString = 'Message ' + messageIdx + ' from connection #' +
+        connection.connectionId + ': ' + data.message;
+    addMessage(logString, data.lang);
+    const fruit = data.message.toLowerCase();
```
Comment (markafoltz) [inline presentation-api/receiver/receiver.js]: Might help to factor out a maybeSetFruit(data.message) function here.

File: presentation-api/cast.html [hunk 2]
```diff
@@ -17,97 +17,86 @@ <h3>Background</h3>
...
+</script>
+
 {% capture js %}
 // App ID EF1A139F is registered in the Google Cast SDK Developer
 // Console and points to the following custom receiver:
 // https://googlechrome.github.io/samples/presentation-api/receiver/index.html
-var presentationUrl = 'cast:EF1A139F';
-var presentationRequest;
-var presentationConnection;
+const presentationRequest = new PresentationRequest('cast:EF1A139F');
```
Comment (markafoltz) [inline presentation-api/cast.html:35]: Can you set navigator.presentation.defaultRequest here, so this presentation is the default choice when casting from the toolbar/menu?

File: presentation-api/index.html [hunk 3]
```diff
@@ -41,122 +37,97 @@ <h3>Background</h3>
...
-{% endcapture %}
 {% include output_helper.html initial_output_content=initial_output_content %}
 
+<script>
+  log = ChromeSamples.log;
+</script>
+
 {% capture js %}
-const presentationUrls = ['receiver/index.html'];
+const presentationRequest = new PresentationRequest(['receiver/index.html']);
```
Comment (markafoltz) [inline presentation-api/index.html:57]: Can you set navigator.presentation.defaultRequest here, so this presentation is the default choice when casting from the toolbar/menu?

File: presentation-api/cast.html [hunk 4]
```diff
@@ -17,97 +17,86 @@ <h3>Background</h3>
...
-    document.querySelector('#terminate').disabled = true;
-  }
+  log('Terminating connection...');
+  presentationConnection.terminate();
+});
+
+document.querySelector('#reconnect').addEventListener('click', () => {
+  const presentationId = document.querySelector('#presentationId').value.trim();
+
+  presentationRequest.reconnect(presentationId)
```
Comment (markafoltz) [inline presentation-api/cast.html]: reconnect() may not work for Cast URLs with the presentationConnection.id as given...

**Review by markafoltz — COMMENTED** [review]
Thanks for the improvements! This looks nicer.
A few comments but fine if you would like me to merge now.

File: presentation-api/cast.html (hunk 2 above)
Comment (beaufortfrancois) [reply presentation-api/cast.html:35]: Done.

File: presentation-api/cast.html (hunk 4 above)
Comment (beaufortfrancois) [reply presentation-api/cast.html]: I've removed it then. Thanks!

File: presentation-api/receiver/receiver.js (hunk 1 above)
Comment (beaufortfrancois) [reply presentation-api/receiver/receiver.js]: Done.

File: presentation-api/index.html (hunk 3 above)
Comment (beaufortfrancois) [reply presentation-api/index.html:57]: Done.

**Comment by beaufortfrancois** [comment]
@mfoltzgoogle I've addressed your feedback at https://github.com/GoogleChrome/samples/pull/564/commits/99e2b3fe78e16dc6c46befa74d8e65bad9f4e9df.

Updated live previews:
https://beaufortfrancois.github.io/samples/presentation-api/
https://beaufortfrancois.github.io/samples/presentation-api/cast.html

**Review by markafoltz — APPROVED** [review]
LGTM!  Thanks!

**Pull request merged**